.pre-commit-config.yaml

```diff
@@ -4,6 +4,6 @@ repos:
     hooks:
       - id: flake8
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.740
+    rev: v0.812
```

Comment:
This has snuck in as needed this version for all type checks to pass. I have just enabled the pre-commit hooks locally, `mypy==0.812` was already installed locally (In other words, the code has been failing `mypy` checks for some time on `v0.740`, this isn't something new introduced by this PR). 

@jklaise if you are bothered, I can redo this as a separate commit/PR...

Reply:
Great, can you updated the lower bound in https://github.com/SeldonIO/alibi-detect/blob/5927d5b4bd8a4f7209cd42db344145f734074d6b/requirements/dev.txt#L4 please?